src/Xamarin.Android.Build.Tasks/Tasks/BuildAppBundle.cs

```diff
@@ -78,13 +81,34 @@ public override bool RunTask ()
 						uncompressed.Add ("**/*" + extension);
 					}
 				}
-				var json = JsonConvert.SerializeObject (new {
+
+				var json = JObject.FromObject(new { });
+				if(!string.IsNullOrWhiteSpace(CustomBuildConfigFile) && File.Exists(CustomBuildConfigFile))
+				{
+					try
+					{
+						using (StreamReader file = File.OpenText(CustomBuildConfigFile))
+						using (JsonTextReader reader = new JsonTextReader(file))
+						{
+							json = (JObject)JToken.ReadFrom(reader);
+						}
+					}
+					catch (Exception e)
+					{ } // Do nothing for now
```

Comment:
If there is an exception we should at least log a warning via `Log.LogCodedWarning` with the details of the exception.

Reply:
You might just let this exception `throw` as we generate a reasonable error code by default now: https://github.com/xamarin/xamarin-android/blob/fbe4b44c7b40dcaacdecaed834c631b15490cc52/src/Xamarin.Android.Build.Tasks/Tasks/AndroidTask.cs#L18-L21

Reply:
Fixed in 478cf6d